From 662f9c2cbd78e9f3f1f5adf696828439ff71a385 Mon Sep 17 00:00:00 2001
From: Rexiel Scarlet <37258415+Rexcrazy804@users.noreply.github.com>
Date: Sat, 13 Jun 2026 12:27:25 +0400
Subject: [PATCH] added mangowc support

---
 conf.d/done.fish | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/conf.d/done.fish b/conf.d/done.fish
index 8160f06..64f968a 100644
--- a/conf.d/done.fish
+++ b/conf.d/done.fish
@@ -87,6 +87,9 @@ function __done_get_focused_window_id
     else if test -n "$NIRI_SOCKET"
         and type -q jq
         niri msg --json focused-window | jq ".id"
+    else if test -n "$MANGO_INSTANCE_SIGNATURE"
+        and type -q jq
+        mmsg get focusing-client | jq '.id'
     else if begin
             test "$XDG_SESSION_DESKTOP" = gnome; and type -q gdbus
         end
-- 
2.54.0

