diff --git a/misc/requirements.in b/misc/requirements.in
index 16200338..b9314ac3 100644
--- a/misc/requirements.in
+++ b/misc/requirements.in
@@ -1,6 +1,5 @@
 setuptools-scm
 requests < 3.0
-PySide6-Essentials
 QtAwesome
 legendary-gl @ https://github.com/derrod/legendary/archive/3963382b3f33116154399be5dd2d2913336a1f0e.zip
 orjson
diff --git a/pyproject.toml b/pyproject.toml
index 011e564d..b3661acc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -23,7 +23,6 @@ classifiers = [
 requires-python = ">= 3.9"
 dependencies = [
     "requests < 3.0",
-    "PySide6-Essentials >= 6.8.1",
     "QtAwesome",
     "legendary-gl >= 0.20.34",
     "orjson",
@@ -149,7 +148,6 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.9"
 requests = "^2.28.1"
-PySide6-Essentials = "^6.8.1"
 QtAwesome = "^1.1.1"
 legendary-gl = "^0.20.34"
 pywebview = [
@@ -174,4 +172,4 @@ PySide6-stubs = "^6.4.2.0"
 
 [tool.poetry-dynamic-versioning]
 enable = true
-vcs = "git"
\ No newline at end of file
+vcs = "git"
